cdap-ui/test/unit/filters/bytes-test.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+describe('bytes', function() {
+
+  beforeEach(module('cdap-ui.filters'));
+
+  var bytes,
+      $filter;
+
+  beforeEach(inject(function(_$filter_) {
+    $filter = _$filter_;
+  }));
+
+  beforeEach(function() {
+    bytes = $filter('bytes');
```

Comment:
Would it be better if we combined the two ```beforeEach``` calls to one?
Something like this,

```
  var bytes;

  beforeEach(inject(function(_$filter_) {
    bytes = _$filter_('bytes');
  }));
```

One less variable and one less ```beforeEach``` invocation :smile: